CHANGELOG.md

```diff
@@ -1,8 +1,63 @@
+## v0.25.0 (2024-02-06)
+
+> **Note:**
+>  Waku Filter v2 now has three additional configuration options
+> `--filter-max-peers-to-serve=1000` drives how many peers can subscribe at once and
+> `--filter-max-criteria=1000` defines what is the maximum criterion stored per each peers
+>
+> This release introduces a major change in Filter v2 protocol subscription management.
+> From now each subscriber peer needs to refresh its living subscriptions by sending a SUBSCRIBER_PING message every 5 minutes by default, otherwise the peer's subscription will be removed.
```

Comment:
Minor nitpick. Maybe a typo?
```suggestion
> From now each subscribed peer needs to refresh its living subscriptions by sending a SUBSCRIBER_PING message every 5 minutes by default, otherwise the peer's subscription will be removed.
```